include/xtensor/xhistogram.hpp

```diff
@@ -0,0 +1,307 @@
+/***************************************************************************
+* Copyright (c) 2016, Johan Mabille, Sylvain Corlay and Wolf Vollprecht    *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+/**
+ * @brief construct histogram
+ */
+
+#ifndef XTENSOR_HISTOGRAM_HPP
+#define XTENSOR_HISTOGRAM_HPP
+
+#include "xtensor.hpp"
+#include "xsort.hpp"
+
+namespace xt
+{
+
+    /**
+     * @ingroup basic_functions
+     * @brief Compute the histogram of a set of data.
+     *
+     * @param The data.
+     * @param The bin-edges.
+     * @param Weight factors corresponding to each data-point.
+     * @param If true the resulting integral is normalized to 1. [default: false]
```

Comment:
I think you need to name the parameter before documenting them (even if not, I think it's easier to understand)